### PR TITLE
biniou 1.0.12

### PR DESCRIPTION
**Pull request by mjambon**

### CODE REVIEW

**Comment by yallop** [comment]
The `install` step is [failing](https://travis-ci.org/ocaml/opam-repository/jobs/142631893):

```
[ERROR] Installation of biniou.1.0.12 failed
Processing  5/5: [biniou: ocamlfind remove]
#=== ERROR while installing biniou.1.0.12 =====================================#
Some files in ~/.opam/4.03.0/install/biniou.install couldn't be installed:
  - /home/travis/.opam/4.03.0/build/biniou.1.0.12/bdump to /home/travis/.opam/4.03.0/bin
```

File: packages/biniou/biniou.1.0.12/files/biniou.install [hunk 1]
```diff
@@ -0,0 +1 @@
+bin: ["bdump"]
```
Comment (yallop) [inline packages/biniou/biniou.1.0.12/files/biniou.install]: Perhaps this should be something like

```
bin: [
  "?bdump.byte"   {"bdump"}
  "?bdump.native" {"bdump"}
]
```

Comment (mjambon) [inline packages/biniou/biniou.1.0.12/files/biniou.install]: Thanks! The install should work now.

**Comment by yallop** [comment]
Installation is working now, but the [3.12 build is failing](https://travis-ci.org/ocaml/opam-repository/jobs/142813896):

```
# ocamlfind ocamlc -a -g -annot -bin-annot -o biniou.cma \
# 		-package "easy-format" bi_util.mli bi_util.ml bi_share.mli bi_share.ml bi_outbuf.mli bi_outbuf.ml bi_inbuf.mli bi_inbuf.ml bi_vint.mli bi_vint.ml bi_io.mli bi_io.ml bi_dump.ml bi_stream.mli bi_stream.ml
```

**Pull request merged**